<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>820</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>deb-check</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>651</width>
      <height>36</height>
     </rect>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>在此输入crp地址</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_check">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>101</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>check</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_fullPackageName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>551</width>
      <height>331</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_name">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>100</y>
      <width>241</width>
      <height>471</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>551</width>
      <height>91</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>63</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>包名</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>70</y>
      <width>63</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>软件名</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>447</y>
      <width>63</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>安装命令</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_copy">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>440</y>
      <width>106</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>一键复制</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
